pkgs/build-support/bintools-wrapper/ld-wrapper.sh

```diff
@@ -20,15 +20,23 @@ if [ -z "${NIX_BINTOOLS_WRAPPER_FLAGS_SET_@suffixSalt@:-}" ]; then
     source @out@/nix-support/add-flags.sh
 fi
 
-setDynamicLinker=1
 
 # Optionally filter out paths not refering to the store.
 expandResponseParams "$@"
+
+# NIX_LINK_TYPE is set if ld has been called through our cc wrapper
```

Comment:
Maybe something like this?
```suggestion
# NIX_LINK_TYPE is set if ld has been called through our cc wrapper. We take
# advantage of this to avoid both recalculating it, and also repeating other
# processing cc wrapper has already done.
```

Reply:
In fact we cannot even re-calculate it since the flags are not passed through. Binutils has no concept of `-static-pie`.